Sources/Moya/TargetType.swift

```diff
@@ -27,6 +27,9 @@ public protocol TargetType {
 
     /// Whether or not to perform Alamofire validation. Defaults to `false`.
     var validate: Bool { get }
+
+    // The headers to used in the request.
```

Comment:
Just noticed I left a typo here 😄
We should use either `The headers used` or `The headers to be used`